Swiftfin/Views/LibraryListView.swift

```diff
@@ -16,7 +17,16 @@ struct LibraryListView: View {
 	@StateObject
 	var viewModel = LibraryListViewModel()
 
-	let supportedCollectionTypes = ["movies", "tvshows", "boxsets", "other"]
+	@Default(.Experimental.liveTVAlphaEnabled)
+	var liveTVAlphaEnabled
+
+	var supportedCollectionTypes: [String] {
+		if liveTVAlphaEnabled {
+			return ["movies", "tvshows", "livetv", "boxsets", "other"]
```

Comment:
I recommend looking to see if these (and their usage through this file) could be changed to use the `itemType` property declared in `BaseItemDtoExtensions`.

Reply:
Addressed, but it required some minor changes to support creating `itemType` from raw string value.